codecov.yml

```diff
@@ -0,0 +1,17 @@
+coverage:
+  status:
+    project:
+      default:
+        # basic
+        #target: 80%
```

Comment:
@ashtul what will be the behavior change here?

Reply:
This can be removed but left for future tweaking.
Right now, with ForkGC uncovered, we always get an error so we ignore it altogether.
The important line is `Informational : True` which removes the error.

Reply:
@gkorland are we ok with those parameters?

Reply:
These lines can be removed altogether. they are commented out.

Reply:
@ashtul so what does it fix?

Reply:
The `x` on many PRs due to CodeCov